benchmarks/burgers/burgers_package.hpp

```diff
@@ -40,6 +43,10 @@ void lr_to_flux(const Real uxl, const Real uxr, const Real uyl, const Real uyr,
   fuz = 0.5 * (sr * uzl * upl - sl * uzr * upr + sl * sr * (uzr - uzl)) * islsr;
 }
 
+struct Region {
+  std::array<Real, 3> xmin, xmax;
+};
+
```

Comment:
In principle, the `RegionSize` struct could be reused.

Reply:
I know... And I initially planned to do that, but `RegionSize` isn't a `struct` anymore, it's a class with accessors and private scope. (I think this changed in the multigrid PR). It was just more cumbersome than doing my own thing. I'll add a comment explaining.